### DISCUSSION TITLE
computational workflow on multiome data

### DISCUSSION THREAD

**Discussion by tanasa** [GreenleafLab/ArchR#1441] [General]
Hi Ryan, and everyone, 

A question about the down-stream analysis of multiome data. We have followed the tutorial described on the page : 

https://greenleaflab.github.io/ArchR_2020/Ex-Analyze-Multiome.html

In the continuation of the tutorial, when I have called the marker genes per cluster, it was expected to see the same markers that were called when have started from either 1) GeneScoreMatrix, or from 2) GeneExpressionMatrix.

As we have started from Multiome data, we do not need to setup an GeneIntegrationMatrix correct ?

Consequently, after calling the differential markers, shal lwe  re-start the workflow from the section  below, when we work with Multiome data :  https://www.archrproject.com/bookdown/pseudo-bulk-replicates-in-archr.html

Please advise, Thank you.

**Comment by rcorces** [comment]
> it was expected to see the same markers that were called when have started from either 1) GeneScoreMatrix, or from 2) GeneExpressionMatrix.

`GeneScoreMatrix` is not a perfect (nor even a near perfect) estimation of gene expression. These matrices would be expected to be substantially different.

> As we have started from Multiome data, we do not need to setup an GeneIntegrationMatrix correct ?

yes

> Consequently, after calling the differential markers, shal lwe re-start the workflow from the section below, when we work with Multiome data

I dont provide recommendations for individual analyses. You'll have to make decisions for your workflow on your own.
